tools/table_annovar/table_annovar.xml

```diff
@@ -42,7 +42,7 @@
         #set operation = ','.join( $operation )
 
         ## Annotate variants.
-        table_annovar.pl ${tab_anno_input} ${__get_data_table_entry__('annovar_indexes', 'dbkey', $input.dbkey, 'path')} -protocol ${protocol} -operation ${operation} -nastring '.' -buildver ${input.dbkey} --outfile output
+        \${SCRIPT_PATH}/table_annovar.pl ${tab_anno_input} ${__get_data_table_entry__('annovar_indexes', 'dbkey', $input.dbkey, 'path')} -protocol ${protocol} -operation ${operation} -nastring '.' -buildver ${input.dbkey} --outfile output
```

Comment:
Sure, thanks for understanding!
Feel free to drop by https://webchat.freenode.net/?channels=#galaxyproject if you need help figuring this out (and if you see a way to help the documentation that's always welcome as well).